opencti-platform/opencti-graphql/src/domain/stixCoreObject.js

```diff
@@ -190,10 +191,12 @@ export const askElementEnrichmentForConnector = async (context, user, enrichedId
   const message = {
     internal: {
       work_id: work.id, // Related action for history
-      applicant_id: user.id, // User asking for the import
+      applicant_id: null, // No specific user asking for the import
```

Comment:
Shouldn't we set the applicant_id to the connector user id if we want the enrichment update to have the connector as the creator?